Marksman/Server.fs

```diff
@@ -793,7 +793,7 @@ type MarksmanServer(client: MarksmanClient) =
                 monad' {
                     let! folder, doc = State.tryFindFolderAndDoc docUri state
 
-                    let maxCompletions = 50
+                    let maxCompletions = (State.userConfigOrDefault state).ComplCandidates()
```

Comment:
```suggestion
                    let maxCompletions = (Folder.configOrDefault folder).ComplCandidates()
```

The terminology here might be a bit confusing, but `State.userConfig` is whatever is in `default.marksman.toml`. Then each `Folder` has its own `.marksman.toml` config file. With this change everything should work.